### ISSUE TITLE
compilation issues with Mingw64 /gcc 10.2

### ISSUE THREAD

**Issue by sancelot** [flexible-collision-library/fcl#516]
It looks like there are some dllimport and functions may be declared as inline that conflicts

leading to many compiler (gcc10.2 ) messages as this one:

 mingw-w64-libfcl/fcl-0.6.1/include/fcl/math/motion/taylor_model/interval-inl.h:417:13: warning: 'fcl::Interval fcl::bound(const fcl::Interval &, S) **redeclared without dllimport attribute: previous dllimport ignored [-Wattributes]**

**Comment by sherm1** [comment]
We would welcome a PR to fix this!

**Comment by sancelot** [comment]
I agree,  I understand there are some dll imports definitions in headers, that should be defineedd in inl.h files or inlined.... I am not really sure how to solve it at the moment

**Comment by sancelot** [comment]
Sorry, but your help is needed,you must keep in mind that this code is not very easy to understand
 
duplicate of https://github.com/flexible-collision-library/fcl/issues/474

**Comment by jamiesnape** [comment]
It should not really be a duplicate of #474. CMake *should* be handling the right syntax cross-platform. Assuming it is, if this compiler is complaining in this respect, it means a mistake was made by developers for all platforms, but it is less of an issue with those platforms. That being said the CMake is very messy at the moment, so potentially there is an issue, but I think this one may be down to the C++ code.
